### PR TITLE
Add required flags to `terra resource create gcp-notebook` in README

### PR DESCRIPTION
**Pull request by melissachang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -362,8 +362,9 @@ Commands:
 ```
 
 You can create a [GCP Notebook](https://cloud.google.com/vertex-ai/docs/workbench/notebook-solution) controlled
-resource with `terra resource create gcp-notebook`. These `stop`, `start` commands are provided for convenience.
-You can also stop and start the notebook using the `gcloud notebooks instances start/stop` commands.
+resource with `terra resource create gcp-notebook --name=resourcename --workspace=workspaceid`. These `stop`, `start`
```
Comment (yuhuyoyo) [inline README.md:365]: i don't think you need to specify workspace if you've done `terra workspace set --id=workspaceid`

**Review by yuhuyoyo — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (melissachang) [reply README.md]: Added braces to make it optional. (I wanted to include it in case someone hasn't done `terra workspace set --id=workspaceid`.)

**Review by yuhuyoyo — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -362,8 +362,9 @@ Commands:
 ```
 
 You can create a [GCP Notebook](https://cloud.google.com/vertex-ai/docs/workbench/notebook-solution) controlled
-resource with `terra resource create gcp-notebook`. These `stop`, `start` commands are provided for convenience.
-You can also stop and start the notebook using the `gcloud notebooks instances start/stop` commands.
+resource with `terra resource create gcp-notebook --name=resourcename [--workspace=workspaceid]`. These `stop`, `start`
```
Comment (marikomedlock) [inline README.md:365]: nit: Recommend wrapping the parameter values in `<>` to be consistent with CLI help format. `--name=<resourcename> [--workspace=<id>]`.

**Review by marikomedlock — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (melissachang) [reply README.md]: Done

**Pull request merged**